tests/framework/utils/k8s_helper.go

```diff
@@ -441,6 +441,48 @@ func (k8sh *K8sHelper) GetPodDetails(podNamePattern string, namespace string) (s
 	return strings.TrimSpace(result), nil
 }
 
+//GetPodEvents returns events about a pod
+func (k8sh *K8sHelper) GetPodEvents(podNamePattern string, namespace string) (*v1.EventList, error) {
+	uri := fmt.Sprintf("api/v1/namespaces/%s/events?fieldSelector=involvedObject.name=%s,involvedObject.namespace=%s", namespace, podNamePattern, namespace)
```

Comment:
you have to use the REST api here? i would have hoped there would be a stronger api for events

Reply:
there is one but i couldnt figure out how to get the registered scheme for unmarshalling the object. I would have to refactor a lot of the integration testing code to get it..